src/pages/Possibility/TwelveDaysCalendar.vue

```diff
@@ -0,0 +1,469 @@
+<template>
+	<div>
+		<ol class="reward-list row">
+			<li
+				v-for="(reward, index) in rewards"
+				:key="`day-${index}`"
+				class="columns small-12 medium-6 large-4 xxlarge-3"
+			>
+				<div
+					v-if="isPast(index)"
+					class="reward reward--past hide-for-small-only"
+					:class="rewardColorClass(index)"
+				>
+					<div class="folded-corner"></div>
+					<component :is="numberComponent(index)" class="number" />
+				</div>
+				<button
+					v-else-if="isPresent(index)"
+					class="reward reward--present"
+					:class="rewardColorClass(index)"
+					@click="showReward"
+				>
+					<SquigglyLine class="squiggly-line" />
+					<StarBig class="star-big" />
+					<StarLittle class="star-little" :class="starPositionClass(index)" />
+					<component :is="numberComponent(index)" class="number" />
+					<router-link
+						:to="reward.url"
+						class="reward-link"
+					>
+						<p class="body-text">
+							Today: {{ reward.text }}
+						</p>
+						<div class="body-cta">
+							{{ reward.ctaText }}
+						</div>
+					</router-link>
+				</button>
+				<div v-else class="reward" :class="rewardColorClass(index)">
+					<SquigglyLine class="squiggly-line" />
+					<StarLittle class="star-little" :class="starPositionClass(index)" />
+					<component :is="numberComponent(index)" class="number" />
+				</div>
+			</li>
+		</ol>
+
+		<svg width="0" height="0">
+			<defs>
+				<linearGradient id="shinygold" x1="0%" y1="0%" x2="100%" y2="160%">
+					<stop offset="0%" stop-color="#d1ba6e" />
+					<stop offset="20%" stop-color="#ebe080" />
+					<stop offset="40%" stop-color="#d1ba6e" />
+					<stop offset="60%" stop-color="#ebe080" />
+					<stop offset="80%" stop-color="#d1ba6e" />
+					<stop offset="100%" stop-color="#ebe080" />
+				</linearGradient>
+			</defs>
+		</svg>
+	</div>
+</template>
+
+<script>
+import Number1 from '@/assets/inline-svgs/12-days-of-lending/1.svg';
+import Number2 from '@/assets/inline-svgs/12-days-of-lending/2.svg';
+import Number3 from '@/assets/inline-svgs/12-days-of-lending/3.svg';
+import Number4 from '@/assets/inline-svgs/12-days-of-lending/4.svg';
+import Number5 from '@/assets/inline-svgs/12-days-of-lending/5.svg';
+import Number6 from '@/assets/inline-svgs/12-days-of-lending/6.svg';
+import Number7 from '@/assets/inline-svgs/12-days-of-lending/7.svg';
+import Number8 from '@/assets/inline-svgs/12-days-of-lending/8.svg';
+import Number9 from '@/assets/inline-svgs/12-days-of-lending/9.svg';
+import Number10 from '@/assets/inline-svgs/12-days-of-lending/10.svg';
+import Number11 from '@/assets/inline-svgs/12-days-of-lending/11.svg';
+import Number12 from '@/assets/inline-svgs/12-days-of-lending/12.svg';
+import SquigglyLine from '@/assets/inline-svgs/12-days-of-lending/squiggly_line.svg';
+import StarBig from '@/assets/inline-svgs/12-days-of-lending/star_big.svg';
+import StarLittle from '@/assets/inline-svgs/12-days-of-lending/star_little.svg';
+
+export default {
+	components: {
+		Number1,
```

Comment:
Currently using a [special webpack loader](https://github.com/visualfanatic/vue-svg-loader) to inline the SVGs instead of <img src=".svg" them. They need to be inlined so the fill colors can be set with CSS. Not sure this is the best way to get it done though. Would love some feedback.

Reply:
This approach is fine by me

Reply:
Yea we probably could have made it work with a similar approach to the icons but I don't want all of this loaded everywhere.

I also really like the added capability of including and using an svg ONLY where we need it.

Reply:
I like this approach for svgs that we want to be able to change/animate in the page